raw-mm.install.lua

```diff
@@ -972,27 +996,29 @@ function installstart(fresh)
   install.ids = {}
   local ids = install.ids
 
-  if not sys.enabledgmcp then
+  --gmcp implimentation needs work and thought so sticking with the non-gmcp implimentation here for now. 
+  --if not sys.enabledgmcp then
+  if ids then
     for skill, skilldata in pairs(installdata) do
       ids[skill] = {}
       if skilldata.yes then ids[skill][#ids[skill]+1] = tempRegexTrigger(skilldata.yes.pattern, skilldata.yes.script) end
       if skilldata.no then ids[skill][#ids[skill]+1] = tempRegexTrigger(skilldata.no.pattern, skilldata.no.script) end
       if skilldata.other then ids[skill][#ids[skill]+1] = tempRegexTrigger(skilldata.other.pattern, skilldata.other.script) end
       send(skilldata.command, false)
     end
-  else
-    for skill, skilldata in pairs(installdata) do
-      if skilldata.gmcp then
-        sendGMCP("Char.Skills.Get "..yajl.to_string(skilldata.gmcp))
-        ids[skill] = true
-      end
-    end
+--   else
+--     for skill, skilldata in pairs(installdata) do
```

Comment:
Let's delete it for now, can always pull from git history later.

Reply:
done